plugins/nextcloud/js/addressbook.js

```diff
@@ -0,0 +1,38 @@
+(rl => {
+	if (rl) {
+		addEventListener('rl-view-model', e => {
+			if ('SettingsContacts' === e.detail.viewModelTemplateID) {
+				const container = e.detail.viewModelDom.querySelector('.form-horizontal');
+				if (container) {
+					rl.pluginRemoteRequest((iError, oData) => {
+						if (!iError) {
+							const mainDivElement = Element.fromHTML('<div class="control-group">'
+								+ '<label>' + rl.i18n("NEXTCLOUD/ADDRESS_BOOK") + '</label>'
+								+ '</div>');
+
+							const selectElement = Element.fromHTML('<select></select>');
+
+							const books = JSON.parse(oData.Result.addressbooks);
```

Comment:
@fahim44 we can use better const name as books not looks descriptive to me. Wdyt about `const addressbooks`?

Reply:
@avinash-0007 please check again